Update Cloud/Portal articles (#2410)
umbraco/UmbracoDocs: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md

By default the projects are ordered by latest updates - projects you've worked on recently will be shown on top and the ones you haven't worked with for a while are in the bottom.

For quick access you can *favorite* the projects you are currently working on, to make them appear as the first projects in the view. You can also browse through the projects by using the search field.

In the top-right corner of the projects view you'll find an option to create more projects. 

Choose the plan you want and a project will be ready for you within a few minutes.

You can also see notifications for your different projects in the top-right corner,

> e.g if your project have been automaticly updated or if the upgrade failed.

Reviewer: ```suggestion
e.g if your project have been automatically updated or if an upgrade has failed.
```

In the bottom-right corner of the Umbraco Cloud Portal, you'll find a chat-bubble. This is where you will be able to reach out to the Umbraco HQ Support Warriors should you have any questions regarding your Umbraco Cloud projects.

> If you are on a professional plan you are entitled to support through the chat regarding implementation and issues with the CMS.

Reviewer: ```suggestion
If you are on a Professional plan you are entitled to support through the chat regarding implementation and issues with the CMS.
```

> With the Starter and standard plan you are only entitled to support regarding specific issues on Cloud. 

Reviewer: ```suggestion
With the Starter and Standard plan you are only entitled to support regarding specific issues regarding the Cloud platform. 
```

This email address is used for logging in to Umbraco Cloud and will receive email notifications from the Umbraco Cloud Portal.

> Note that it is not possible to change email at a later point.

Reviewer: ```suggestion
Note that it is not possible to change this email at a later point.
```
